src/registrar/templates/includes/footer.html

```diff
@@ -0,0 +1,123 @@
+{% load static %}
+
+<footer class="usa-footer">
+    <div class="usa-footer__secondary-section">
+      <div class="grid-container">
+        <div class="grid-row grid-gap">
+          <div
+            class="
+              usa-footer__logo
+              grid-row
+              mobile-lg:grid-col-6 mobile-lg:grid-gap-2
+            "
+          >
+            <div class="mobile-lg:grid-col-auto">
+              <img
+                class="usa-footer__logo-img"
+                src="{% static 'img/dottedgov-round.svg' %}"
+                alt="dot gov registrar"
```

Comment:
**Suggestion(non-blocking)**: Either change the alt text to read "dot gov registrar logo" or set the alt text to blank (`alt=""`). [Longer reading about alt text](https://webaim.org/techniques/alttext/).

Reply:
Changed to include "logo" to match the "CISA logo" nearby.